UnrealPlugin/ThirdPersonMPGSDKSetup.md

```diff
@@ -0,0 +1,316 @@
+# ThirdPersonMP Example - GSDK Project Setup
+
+This guide covers upgrading the finished Unreal ThirdPersonMP template and tutorial project, by adding and configuring the PlayFab GSDK. By following this guide, you will upgrade your existing project so it can be hosted on PlayFab Multiplayer Services.
+
+Please note, your Unreal project must have the following capabilities:
+
+* Networking
+* Multiplayer
+* Dedicated Game-Server
+
+If not, you will need to return to the [ThirdPersonMP Example Project Setup](ThirdPersonMPSetup.md) guide to configure your project as a network-enabled multiplayer game, with dedicated server capabilities.
+
+## Goals
+
+We will add and configure the Playfab Unreal GSDK to your project, and test it locally to verify it is expected to work on PlayFab Multiplayer Services.
+
+## Requirements
+
+* Download Visual Studio (the [community version is free](https://visualstudio.microsoft.com/vs/community/))
+	* Requires Workloads: .NET desktop development, and Desktop development with C++
+* Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal Engine website. This was tested on [4.26.2](https://github.com/EpicGames/UnrealEngine/releases/tag/4.26.2-release) (Link requires permissions)
+* A completed [ThirdPersonMP Example Project](ThirdPersonMPSetup.md), or a project with similar capabilities
+* [PlayFab Unreal GSDK plugin](https://github.com/PlayFab/gsdk/tree/master/UnrealPlugin)
+* [Optional] [PlayFab Marketplace plugin](https://www.unrealengine.com/marketplace/en-US/product/playfab-sdk) or the [source version on GitHub](https://github.com/PlayFab/UnrealMarketplacePlugin/tree/master/4.26/PlayFabPlugin/PlayFab) (Not required for GSDK, but required for most other PlayFab services, including login)
+
+## C++ Implementation
+
+### Adding the plugin to the project
+
+When using the plugin a few things need to be taken care off.
+
+First, open File Explorer and create a folder called "Plugins" in your games' root directory and in the Plugins folder, create a folder called "PlayFabGSDK". Then, drag all the files from the UnrealPlugin folder in this repo into the Plugins/PlayFabGSDK folder.
+
+Open the .uproject file in a text editor of your choice. In the plugins array add the PlayFabGSDK.
```

Comment:
"PlayFabGSDK plugin" ??

Reply:
I'm going to disagree with this suggestion. The word plugin is already present in the sentence, and the code example below is clear.

Reply:
on a second read, you're right. Ignore